### PR TITLE
TEC 286 - Blog Post Sharebar 

### PR DESCRIPTION
**Pull request by gcedo**
Lives @ http://gcedo.github.io/component-blog-post/

### CODE REVIEW

File: assets/icons.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...c7.847-10.325 14.455-7.847 23.954-4.543l68.97 36.344zM512 99a413 413 0 0 1 413 413c0 228.094-184.906 413-413 413-81.36 0-156.94-23.54-220.955-64.015L99 925l64.015-192.045C122.54 668.94 99 593.36 99 512 99 283.906 283.906 99 512 99zm0 82.6c-182.475 0-330.4 147.925-330.4 330.4 0 71.036 22.302 136.703 60.298 190.393L202.25 821.75l119.357-39.648C375.297 820.098 440.964 842.4 512 842.4c182.475 0 330.4-147.925 330.4-330.4 0-182.475-147.925-330.4-330.4-330.4z"/></symbol><symbol viewBox="0 0 40 40" id="worldif"><title>worldif</title><path d="M8 9h2.8v23H8V9zm3.414 23V9h2.798v23h-2.798zM17.22 9h15.775v2.302H20.018V32h-2.8V9zM31.87 21.143H20.638V11.91H33v2.27h-9.568v4.665h8.435v2.298h.005zM20.632 32V21.752h11.235v2.268h-8.435V32h-2.8z"/></symbol><symbol viewBox="0 0 1024 1024" id="youtube"><title>youtube</title><path d="M433.4 688.28V336.72L667.7 512.5 433.4 688.28zm390.5-472.655C800.47 207.812 655.985 200 511.5 200c-144.485 0-288.97 7.422-312.4 14.844C138.182 235.156 121 371.874 121 512.5c0 140.234 17.182 277.344 78.1 297.266C222.53 817.578 367.015 825 511.5 825c144.485 0 288.97-7.422 312.4-15.234C884.818 789.844 902 652.734 902 512.5c0-140.625-17.182-276.953-78.1-296.875z"/></symbol></svg>
```
Comment (fabiosantoscode) [inline assets/icons.svg]: This file shouldn't be committed to git, as it comes from npm-assets. Can you remove it?

Comment (gcedo) [inline assets/icons.svg]: :+1:

**Comment by fabiosantoscode** [comment]
This pull request makes me happy. :rainbow:

File: parts/blog-post-sharebar.es6 [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  if (typeof window !== 'undefined') {
+    const mobileDetector = new MobileDetect(window.navigator.userAgent);
+    isMobile = mobileDetector.mobile() !== null;
+  }
+
+  return (
+    <div className="blog-post__sharebar">
+      {!isMobile && <ShareBar icons={[ 'twitter', 'facebook' ]} />}
+      <Balloon
+        className="blog-post__toggle-share"
```
Comment (fabiosantoscode) [inline parts/blog-post-sharebar.es6]: Whoops, missed this one.

**Pull request merged**